dpc/src/network/testnet2.rs

```diff
@@ -192,9 +197,8 @@ impl Network for Testnet2 {
     type CiphertextIDCRHGadget = BHPCRHGadget<Self::ProgramProjectiveCurve, Self::InnerScalarField, Self::ProgramAffineCurveGadget, 41, 63>;
     type CiphertextID = AleoLocator<<Self::CiphertextIDCRH as CRH>::Output, { Self::RECORD_CIPHERTEXT_ID_PREFIX }>;
```

Comment:
Note to self: I believe I can delete this now, and its corresponding instantiation methods below.